h/views/account_signup.py

```diff
@@ -19,10 +20,13 @@ def _login_redirect_url(request):
 class SignupController:
     def __init__(self, request):
         self.request = request
-        self.schema = schemas.RegisterSchema().bind(request=self.request)
+        orcid = request.session.get("pending_orcid")
```

Comment:
I was wondering how we would achieve this: when we receive the ORCID ID from ORCID we need to somehow pass that through to our create-account page, and then it also needs to be passed with the form submission request when creating the account, and this needs to be done securely (so users cannot spoof ORCID IDs). Will putting it in the session be enough?